packages/docusaurus/src/commands/build.ts

```diff
@@ -110,7 +110,9 @@ export async function build(
   // Remove server.bundle.js because it is useless
   if (serverConfig.output && serverConfig.output.filename) {
     const serverBundle = path.join(outDir, serverConfig.output.filename);
-    fs.existsSync(serverBundle) && fs.unlinkSync(serverBundle);
+    fs.pathExists(serverBundle).then(exist => {
```

Comment:
Note that this is different from before, if that's intentional. Previously, the code won't proceed until L113 is complete, but now L113 is made async and the code will continue even though `fs.pathExists(serverBundle)` hasn't completed. Just pointing it out in case it makes an important difference to the build step.

Reply:
Yes this is intentional :)

Because we want to proceed to next statement as fast as possible. Actually deleting it or not doesnt matter >.<. Its just nicer to remove this bundle from `build`